src/resources/formats/html/pandoc/template.html

```diff
@@ -36,6 +36,8 @@
 $title-block.html()$
 $elseif(subtitle)$
 $title-block.html()$
+$elseif(by-author)$
```

Comment:
I am thinking that this approach would screen out cases where there is no title, no subtitle, and author normalization is disabled, but the same point could be made about including a date (e.g. user put only a date and it didn't appear).

I would suggest we should just always include `title-block` without condition - what do you think?

Reply:
Makes sense, I made a small change in that direction: cfa2917b120af88bca230298fa9cb5bcdc80913a

Reply:
I made the change 1704e9aec878dec9cf91241af83c18e195a9b6ae and did few manual testing on simple HTML document.
Seems good.